### PR TITLE
[SD-migration] LPS-99260 screens-service

### PR DESCRIPTION
**Pull request by ChiZelin**

### CODE REVIEW

**Comment by ChiZelin** [comment]
ci:test:sf

**Comment by ChiZelin** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
Just started reviewing :)

File: modules/apps/screens/screens-service/src/main/java/com/liferay/screens/service/impl/ScreensAssetEntryServiceImpl.java [hunk 1]
```diff
@@ -50,10 +51,9 @@
 import com.liferay.portal.kernel.security.permission.resource.ModelResourcePermissionFactory;
 import com.liferay.portal.kernel.util.GetterUtil;
 import com.liferay.portal.kernel.util.HtmlUtil;
-import com.liferay.portal.kernel.util.PortalUtil;
+import com.liferay.portal.kernel.util.Portal;
```
Comment (dantewang) [inline modules/apps/screens/screens-service/src/main/java/com/liferay/screens/service/impl/ScreensAssetEntryServiceImpl.java]: @ChiZelin Do you have any specific reason to change from using `PortalUtil` to `Portal` reference? If not then please exclude this change and resend. Thanks

Comment (dantewang) [reply modules/apps/screens/screens-service/src/main/java/com/liferay/screens/service/impl/ScreensAssetEntryServiceImpl.java]: @ChiZeLin The reason is, this ticket is to migrate services from spring to DS. Let's focus only on the migration, and don't touch things unless they are completely wrong.

Comment (ChiZelin) [reply modules/apps/screens/screens-service/src/main/java/com/liferay/screens/service/impl/ScreensAssetEntryServiceImpl.java]: > @ChiZelin Do you have any specific reason to change from using `PortalUtil` to `Portal` reference? If not then please exclude this change and resend. Thanks

@dantewang Yes, I do. Source formatter tells me to do this. And the [example PR](https://github.com/brianchandotcom/liferay-portal/pull/72209/commits/6339cd84dfc7ac7b0efefd87577f1b0cdee88551#diff-a5276e9721732e5419b5f8f32e161cb5L147) mentioned in the email also did this.

**Comment by dantewang** [comment]
Just started reviewing :)

**Comment by dantewang** [comment]
Pull request submitted to https://github.com/Preston-Crary/liferay-portal/pull/836.

**Pull request closed without merging**